internal/runner/runner.go

```diff
@@ -114,11 +114,11 @@ func (r *CommandRunner) runStep(ctx context.Context, step *CommandStep, dry bool
 	if err != nil {
 		return nil, err
 	}
-	posixEnv = append(posixEnv, "PWD="+pwd)
+	// posixEnv = append(posixEnv, "PWD="+pwd)
```

Comment:
It might be a good idea to pass through the `PATH` environment variable here otherwise builders would need to set it.
```suggestion
	posixEnv = append(posixEnv, "PWD="+pwd)
	posixEnv = append(posixEnv, "PATH="+os.Getenv("PATH"))
```